runtime/compiler/z/codegen/S390PrivateLinkage.hpp

```diff
@@ -128,9 +128,9 @@ class PrivateLinkage : public J9::PrivateLinkage
    };
 
 
-////////////////////////////////////////////////////////////////////////////////
+
 //  J9::Z::HelperLinkage Definition for J9
-////////////////////////////////////////////////////////////////////////////////
+
```

Comment:
Lines that contain only slashes will be ignored by Doxygen, so there is no need to change these lines. May I ask you to change these comments back to the following?

```
////////////////////////////////////////////////////////////////////////////////
//  J9::Z::HelperLinkage Definition for J9
////////////////////////////////////////////////////////////////////////////////
```